system/ee/EllisLab/ExpressionEngine/Cli/Commands/CommandPrepareUpgrade.php

```diff
@@ -0,0 +1,453 @@
+<?php
+
+namespace EllisLab\ExpressionEngine\Cli\Commands;
+
+use EllisLab\ExpressionEngine\Cli\Cli;
+use EllisLab\ExpressionEngine\Cli\Commands\Upgrade\UpgradeMap;
+use EllisLab\ExpressionEngine\Library\Filesystem\Filesystem;
+
+class CommandPrepareUpgrade extends Cli
+{
+
+    /**
+     * name of command
+     * @var string
+     */
+    public $name = 'Prepare Upgrade';
+
+    /**
+     * signature of command
+     * @var string
+     */
+    public $signature = 'prepare-upgrade';
+
+    /**
+     * Public description of command
+     * @var string
+     */
+    public $description = 'Prepare a different site to be upgraded using these files';
+
+    /**
+     * Summary of command functionality
+     * @var [type]
+     */
+    public $summary = 'This command copies all files necessary for upgrading into a different ExpressionEngine site and restructures it';
+
+    /**
+     * How to use command
+     * @var string
+     */
+    public $usage = 'php eecli prepare-upgrade';
+
+    /**
+     * options available for use in command
+     * @var array
+     */
+    public $commandOptions = [
+        'upgrade-ee'                => 'Start the upgrade after moving files',
+        'force-add-on-upgrade'      => 'After upgrading EE, runs addon upgrades',
+        'old-base-path:'            => 'Absolute path of old site',
+        'new-base-path:'            => 'Absolute path of new site',
+        'old-public-path:'          => 'Absolute path of old site public path',
+        'new-public-path:'          => 'Absolute path of new site public path',
+        'no-config-file'            => 'Ignores the config file and doesn\'t check for it',
+        'ee-version'                => 'The current site ',
+        'should-move-system-path'   => 'Whether the upgrade process should move the old theme folder to the new site',
```

Comment:
This says "theme" but should say "system"

Reply:
Fixed in latest commit.